sdcm/utils/thread.py

```diff
@@ -107,7 +109,7 @@ def format_error(exc):
         :param exc: the exception
         :return: string to add to the event
         """
-        #  pylint: disable=no-member
+        #  pylint: disable=all
```

Comment:
why did you disabled those checked ?

Reply:
19:45:26  ************* Module multiple_dc_test
19:45:26  multiple_dc_test.py:1:0: R0801: Similar lines in 2 files
19:45:26  ==sdcm.stress_thread:167
19:45:26  ==sdcm.utils.thread:112
19:45:26                  if hasattr(exc, 'result') and exc.result.failed:
19:45:26                      stderr = exc.result.stderr
19:45:26                      if len(stderr) > 100:
19:45:26                          stderr = stderr[:100]
19:45:26                      errors_str = f'Stress command completed with bad status {exc.result.exited}: {stderr}'
19:45:26                  else:
19:45:26                      errors_str = f'Stress command execution failed with: {str(exc)}' (duplicate-code)
19:45:26  
19:45:26  --------------------------------------------------------------------
19:45:26  Your code has been rated at 10.00/10 (previous run: 10.00/10, -0.00)

Unfortunately there is some kind of problem around disabling duplicate-code, disable all handlers is only thing that helped.

Reply:
I agree it's weird it's jumps out in other PRs in not the one initiated it (or other PRs that pass just fine), and that duplicate-code can't be disabled, only per file or global (it's a bug in pylint)

but doing `#  pylint: disable=all` beats the whole point of using pylint... 

please remove all the duplicate-code comment that doesn't actually work.
just change the code a bit, or refactor into a function. (I know it's my fault, that we have this duplicate code, if you want I can do that in a separate PR)

Reply:
this should solve the duplicate-code
https://github.com/scylladb/scylla-cluster-tests/pull/1830

Reply:
Ok, i fall it back.